config/config.go

```diff
@@ -18,13 +18,20 @@ const (
 	ENV_IDENTITY_CERT_FILE        = "IDENTITY_CERT_FILE"
 	ENV_IDENTITY_PRIVATE_KEY_FILE = "IDENTITY_PRIVATE_KEY_FILE"
 
+	ENV_PROMETHEUS_SERVICE = "PROMETHEUS_SERVICE"
+
 	ENV_SERVER_ADDRESS                       = "SERVER_ADDRESS"
 	ENV_SERVER_PORT                          = "SERVER_PORT"
 	ENV_SERVER_CREDENTIALS_USERNAME          = "SERVER_CREDENTIALS_USERNAME"
 	ENV_SERVER_CREDENTIALS_PASSWORD          = "SERVER_CREDENTIALS_PASSWORD"
 	ENV_SERVER_STATIC_CONTENT_ROOT_DIRECTORY = "SERVER_STATIC_CONTENT_ROOT_DIRECTORY"
 )
 
+// Store SWS Config.
+// Access to the config can be necessary to other areas and I guess is better to have a global variable instead of
+// passing the config in the function signature as it can be harder to follow.
+var SWSConfig *Config
```

Comment:
Right, here I would prefer passing it around if at all possible. Global variables might just cause us problems in the future with circular dependencies in imports. If you haven't hit that kind of problem in Go, you will, and its a pain to work around. That's why I avoid global variables and even global functions as much as I can.

Reply:
I commented in the previous annotation, I guess is related same discussion.